lib/web/integrations_awsoidc.go

```diff
@@ -324,6 +324,36 @@ func (h *Handler) awsOIDCConfigureEICEIAM(w http.ResponseWriter, r *http.Request
 	return nil, trace.Wrap(err)
 }
 
+// awsOIDCConfigureAppAccessIAM returns a script that configures the required IAM permissions to enable the usage of App Access
+// using the AWS OIDC Credentials.
```

Comment:
nit: Document inputs and outputs for the endpoint?

Reply:
It has not input and the output is the script

> ... returns a script that ...

Reply:
Query param "role" is a required input, no?

Does it return the script as an attachment or as a literal script? Maybe mention that and the Content-Type used?